docs/generators/java.md

```diff
@@ -91,6 +91,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |useRxJava3|Whether to use the RxJava3 adapter with the retrofit2 library. IMPORTANT: This option has been deprecated.| |false|
 |useSingleRequestParameter|Setting this property to true will generate functions with a single argument containing all API endpoint parameters instead of one argument per parameter. ONLY jersey2, jersey3, okhttp-gson support this option.| |false|
 |webclientBlockingOperations|Making all WebClient operations blocking(sync). Note that if on operation 'x-webclient-blocking: false' then such operation won't be sync| |false|
+|withAWSV4Signature|whether to include AWS v4 signature support| |false|
```

Comment:
@outscale-mdr tested locally and the result is good.

can you update the description to indicate this option only works with `okhttp-gson` library only? then I'll merge.

Reply:
Thank you @wing328  for your time !